core/service/src/backup/tests.rs

```diff
@@ -592,23 +573,24 @@ fn operator_handle_init(
     custodian_count: usize,
 ) -> (
     BTreeMap<
-        Role,
+        Address,
```

Comment:
Why is the address used here instead of the verification key itself like in other files?
I agree both work. But I might be more inclined to the verification key since then no changes will be needed in case we change the signing scheme

Reply:
I think in this case `verf_key_id()` would be more suitable as it is meant to be more general than the `Address`